mysql-test/suite/innodb/t/log_inplace_alter_with_exclusiv_lock.test

```diff
@@ -0,0 +1,33 @@
+--source include/have_debug.inc
+
+--echo #
+--echo # Bug#106163 Inplace Add index with lock=exclusive doesn't generate MLOG_ADD_INDEX redo
+ --echo #
+
+SET GLOBAL innodb_page_cleaner_disabled_debug=1;
+
+--source include/no_checkpoint_start.inc
```

Comment:
instead of checkpoint_end.inc and no_checkpoint, can we use disable checkpoitn_disabled=true?

Reply:
`innodb_checkpoint_disabled`  is not defined in 5.7

Reply:
ok, please take care of this in 8.0 port